Updated WMCO 5.0 Platform Support Tables (#45958)
openshift/openshift-docs: windows_containers/index.adoc

{productwinc} is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. With a Red Hat subscription, you can get support for running Windows workloads in {product-title}. For more information, see the xref:../windows_containers/windows-containers-release-notes-5-x.adoc#windows-containers-release-notes-5-x[release notes].

> Windows nodes can be added either by creating a xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.adoc#creating-windows-machineset-aws[MachineSet] or by specifying existing Bring-Your-Own-Host (BYOH) Window instances through a xref:../windows_containers/byoh-windows-instance.adoc#byoh-windows-instance[configuration map].

Reviewer: ```suggestion
Windows nodes can be added either by creating a xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.adoc#creating-windows-machineset-aws[MachineSet] or by specifying existing Bring-Your-Own-Host (BYOH) Window instances through a xref:../windows_containers/byoh-windows-instance.adoc#byoh-windows-instance[ConfigMap].
```

> Windows nodes can be added either by creating a xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.adoc#creating-windows-machineset-aws[machine set] or by specifying existing Bring-Your-Own-Host (BYOH) Window instances through a xref:../windows_containers/byoh-windows-instance.adoc#byoh-windows-instance[configuration map].

Reviewer 2: ```suggestion
You can add Windows nodes either by creating a xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.adoc#creating-windows-machineset-aws[machine set] or by specifying existing Bring-Your-Own-Host (BYOH) Window instances through a xref:../windows_containers/byoh-windows-instance.adoc#byoh-windows-instance[configuration map].
```

Author: And that one, too. Grrrrrr!

[NOTE]
====

> BYOH is not supported for bare metal/provider agnostic clusters.

Reviewer 3: This should be the opposite: MachineSet nodes are not supported for bare metal/provider agnostic clusters. Only BYOH are (by definition of those install types)

Reviewer: +1

> Machne sets are not supported for bare metal/provider agnostic clusters.

Reviewer 2: ```suggestion
Machine sets are not supported for bare metal or provider agnostic clusters.
```

Author: @kalexand-rh Sorry! I must have missed pushing that change. Thanks!

Reviewer 2: No worries! I'm glad you were on it!
